test/test_datatable.py

```diff
@@ -48,3 +50,46 @@ def test_dataset_column_names_match_expected(self):
         metadata = Datatable('ZACKS/FC').data_fields()
         six.assertCountEqual(self,
                              metadata, [u'datatable_code', u'id', u'name', u'vendor_code'])
+
+
+class BulkDownloadDataTableTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        httpretty.enable()
+        httpretty.register_uri(httpretty.GET,
+                               re.compile(
```

Comment:
Is there any reason for such formatting?

Reply:
not sure, I was looking at the old tests...